compact/far_forward/electron_beamline.xml

```diff
@@ -0,0 +1,116 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
+<!-- Copyright (C) 2022 Jaroslav Adam, Jakub Ceska -->
+
+<lccdd>
+
+  <define>
+
+    <!-- electron beam pipe wall thickness at positive z -->
+    <constant name="cen_pos_wall_dr" value="1.0*mm"/>
+
+    <!-- end of central section -->
+    <constant name="cen_pos_zmax" value="4484.25*mm"/>
+    <constant name="cen_pos_r1" value="BeampipeOD/2."/>
```

Comment:
Can we use more descriptive variable names here?

Reply:
I will try and rename the variables.

Reply:
Implemented